sample/scripts/initSampleCode.ts

```diff
@@ -0,0 +1,44 @@
+export default function initSampleCode() {
+    insertHtmlSampleCode();
+    insertJsSampleCode();
+
+    var fileref=document.createElement('script');
+    fileref.setAttribute("type","text/javascript");
+    fileref.setAttribute("src", "https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js");
```

Comment:
This file is under Apache license. Can we use it here?

Reply:
Looked up online and the Apache license documentation. It should be fine to use Apache licensed work and redistribute in any license as long as we retain any original Apache License notices in source files of the original project.